src/app/select-trajectory/select-trajectory.page.html

```diff
@@ -28,6 +28,7 @@
 
       <ion-col sizeXs="12" sizeLg="4">
         <app-item-card
+          [disabled]="!locationService.isSupportedPlatform"
           icon="add-outline"
           title="Add your own trajectory"
           subtitle="import a GPX or GeoJSON file"
```

Comment:
As your implementation only supports our JSON encoding (which is fine), change this
```suggestion
          subtitle="import a GPX or GeoJSON file"
          subtitle="import a trajectory JSON file"
```